.eslintrc.json

```diff
@@ -0,0 +1,85 @@
+{
+	"root": true,
+	"extends": [
+	  "eslint:recommended",
+	  "plugin:prettier/recommended",
+	  "plugin:@typescript-eslint/eslint-recommended",
+	  "plugin:@typescript-eslint/recommended"
+	],
+	"ignorePatterns": "lib/**/*",
+	"env": {
+	  "node": true,
+	  "mocha": true,
+	  "es6": true
+	},
+	"parserOptions": {
+	  "ecmaVersion": 2019
```

Comment:
2019 seems old for our targets no? I guess bson and driver have 2020 so not too far off.

Reply:
Again, just aligned with mongodb-client-encryption.  Do you want me to adjust it?

Reply:
Fine to leave it as is.